src/HTML.js

```diff
@@ -64,11 +63,10 @@ export default class HTML extends PureComponent {
         baseFontStyle: { fontSize: 14 },
         tagsStyles: {},
         classesStyles: {},
-        textSelectable: false,
-        allowFontScaling: true
+        textProps: {},
     }
 
-    constructor (props) {
+    constructor(props) {
```

Comment:
It seems like your linter or text editor has decided to reformat the entire file. Could you remove any changes that are not related to the proposed feature?